module/data/exam.py

```diff
@@ -81,6 +81,16 @@ def append_session(self, session_name: str, to_append: str):
             # pylint: disable=unnecessary-dunder-call
             self.__getattribute__(session_name).append(to_append)
 
+    @classmethod
+    def append_multiple_sessions(cls, cells, exam, session):
+        for cell in cells:
+            cell_clean_text = cell.text.replace('\xa0', '').replace('\n', '').replace('DMI', '')
+            current_exams = [string.replace('DMI', '') for string in exam.get_session("prima") +
+                             exam.get_session("seconda") + exam.get_session("terza") +
+                             exam.get_session("straordinaria")]
```

Comment:
this is a bit long, isn't it?
```suggestion
            exam_sessions = exam.get_session("prima") + exam.get_session("seconda") + exam.get_session("terza") + exam.get_session("straordinaria")
            current_exams = [string.replace('DMI', '') for string in exam_sessions]
```